content/news/027/index.md

```diff
@@ -88,25 +88,27 @@ small update to improve the UI and fix a few bugs.
 
 ![Soldank](https://raw.githubusercontent.com/smokku/soldank/master/sshot.png)
 
-[Soldank] ([GitHub](https://github.com/smokku/soldank), [Discord](https://discord.gg/cTaC4UtqE6))
-by [@smokku](https://twitter.com/smokkku) is an open source clone of [Soldat] engine.
-It aims for full compatibility with original game files, mods and gameplay with modernized
+[Soldank] ([GitHub](https://github.com/smokku/soldank),
+[Discord](https://discord.gg/cTaC4UtqE6)) by [@smokku](https://twitter.com/smokkku)
+is an open source clone of [Soldat] engine. It aims for full compatibility
+with original game files, mods and gameplay with modernized
 graphics engine and multiplayer networking code.
 
 Recent developments include:
 
-- Engine/Game code split
-- Command Line Interface
+- Engine/game code split
+- Command line interface
```

Comment:
Command Line Interface (capitalized) is a reference to a well-known `CLI` acronym

Reply:
Sure, I can change it back.